### PR TITLE
[IMP] account_banking_mandate: Fill payment line with first valid mandate

### PR DESCRIPTION
**Pull request by carlosdauden**
Without this patch when a customer has more than one partner bank, partner bank and mandate aren't filled on payment line.

https://github.com/OCA/bank-payment/blob/764f22fa14e88c3e82474e92760d6a993c099035/account_payment_order/models/account_move_line.py#L53

@Tecnativa

### CODE REVIEW

File: account_banking_mandate/models/account_move_line.py [hunk 1]
```diff
@@ -16,14 +17,21 @@ class AccountMoveLine(models.Model):
...
-            vals['mandate_id'] = self.mandate_id.id
-            vals['partner_bank_id'] = self.mandate_id.partner_bank_id.id
-        partner_bank_id = vals.get('partner_bank_id', False)
-        if partner_bank_id and 'mandate_id' not in vals:
-            mandate = self.env['account.banking.mandate'].search(
-                [('partner_bank_id', '=', partner_bank_id),
-                 ('state', '=', 'valid')], limit=1)
-            if mandate:
-                vals['mandate_id'] = mandate.id
+        if payment_order.payment_type != 'inbound':
```
Comment (pedrobaeza) [inline account_banking_mandate/models/account_move_line.py:20]: bank account is not set on outbound operations?

Comment (carlosdauden) [reply account_banking_mandate/models/account_move_line.py:20]: Of course, but not in mandate module...
https://github.com/OCA/bank-payment/blob/764f22fa14e88c3e82474e92760d6a993c099035/account_payment_order/models/account_move_line.py#L51-L56

Comment (pedrobaeza) [reply account_banking_mandate/models/account_move_line.py:20]: OK, thanks

**Review by pedrobaeza — APPROVED** [review]

**Review by rafaelbn — APPROVED** [review]

**Comment by rafaelbn** [comment]
This PR need review 👍

**Review by JordiBForgeFlow — APPROVED** [review]
:+1: Technical and functional review

**Comment by JordiBForgeFlow** [comment]
This is good to merge. Perhaps squash the commits to a single one.

**Comment by JordiBForgeFlow** [comment]
I can see that the commits should not be merged. I am merging this PR, and I will cherry-pick to v11

**Pull request merged**